### ISSUE TITLE
[Feature] Compare the Last Remote SHA and Head SHA of a Git Branch?

### ISSUE THREAD

**Issue by vrathore18** [tj-actions/changed-files#1006]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

I have a use case for the "on synchronize" event, which I will illustrate as follows:

I created a pull request with the base branch as "develop" and the current branch as "feature/xyz".
Let's assume the current SHA of the "feature/xyz" branch is SHA 111.
I made changes to "file1.txt" and pushed the changes with a commit SHA of SHA 222.
I was able to detect the change correctly on the "feature/xyz" branch (from SHA 111 to SHA 222).

However, I am facing a challenge now.

Let's assume that I made another change to "file2.txt" on my local machine. I committed my changes but didn't push them (SHA 333).
Then, I made a change to "file3.txt" on my local machine, committed my changes and pushed them (SHA 444).

But when the triggered build checked in, I can only see that the changed file is "file3.txt" because the comparison is between SHA 444 and SHA 333. What I actually need is a comparison between SHA 444 and SHA 222, as SHA 333 was a local commit and came with the SHA 444 push.

### Describe the solution you'd like?

In summary, I want to compare the last remote SHA of the current branch (SHA 222) with the head SHA of the current branch (SHA 444)

It is even possible?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jackton1** [comment]
Hi @vrathore18 I believe this is highlighted in the [Usage](https://github.com/tj-actions/changed-files#usage) documentation.

You can achieve this by 
```
...
	- name: Get changed files
	   id: changed-files
	   uses: tj-actions/changed-files@v35
       with:
          since_last_remote_commit: true
```
